modules/internal/src/main/scala/higherkindness/mu/rpc/internal/encoders/pbd.scala

```diff
@@ -30,7 +30,7 @@ object pbd extends OptionInstances with ListInstances {
 
   import pbdirect._
 
-  implicit def defaultDirectPBMarshallers[A: PBWriter: PBReader]: Marshaller[A] =
+  implicit def defaultDirectPBMarshallers[A: PBMessageWriter: PBReader]: Marshaller[A] =
```

Comment:
This change is quite important. It means it's no longer in valid in Mu to create an RPC endpoint that takes a single field (e.g. a `String`) as a request or return one as a response. You have to use a proper message type (e.g. `case class MyRequest(@pbIndex(1) id: String)` for both requests and responses.

@juanpedromoreno Do you think this is a reasonable change? I found quite a few places in the tests where we were using things like `LocalDateTime` or `BigDecimal` as the RPC request type. I had to delete those tests because they don't compile now.

The only reason for this change is that I did some refactoring in my pbdirect PR, splitting out `PBWriter` into `PBMessageWriter` and `PBFieldWriter`. If necessary I can revert that refactoring and put everything back to how it was before.

Reply:
I think it's definitely a reasonable change 👍